logs.rst

```diff
@@ -1,16 +1,58 @@
-Results while adding implementations -
-===========================
+`Colab Notebook <https://colab.research.google.com/drive/1brU6xL8nqlw5F6op0Edys-Av7SlTrr6t?usp=sharing>`_
```

Comment:
I guess we can make one example notebook later no need to have this in the `logs.rst` file as of now.

Reply:
Removed